app/__init__.py

```diff
@@ -84,6 +84,11 @@ def load_currentTerm():
         session['current_term'] = model_to_dict(term)
         g.current_term = term
 
+import datetime
+@app.before_request
+def load_currentDateTime():
+    g.currentTime = datetime.datetime.now()
```

Comment:
Let's name this variable so that it isn't ambiguous what the data type is. `g.currentDateTime`